src/components/Card/share-asset/ShareAssetD3.tsx

```
@@ -0,0 +1,511 @@
+    placeStamps,
+    placeDecorations,
+    buildStatColumns,
+    usernameFontSize,
+    type StampPlacement,
+    type DecorationPlacement,
+} from './shareAssetLayout'
+import type { ShareAssetD3Props, TierLevel } from './shareAsset.types'
+import { TIER_0_BADGE, TIER_1_BADGE, TIER_2_BADGE, TIER_3_BADGE } from '@/assets/badges'
+import { PEANUTMAN_WAVING, PEANUTMAN_RAISING_HANDS, PEANUTMAN_PFP } from '@/assets/peanut'
+import { STAR_STRAIGHT_ICON } from '@/assets/icons'
+import { HandThumbsUp, Eyes, Sparkle, Cloud } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)